sites/fast-website/src/public/index.html

```diff
@@ -0,0 +1,120 @@
+              <fast-button slot="action" appearance="accent" href="/docs/fast-foundation/getting-started">Get started</fast-button>
+            </site-section-header>
+          </div>
+        </section>
+
+        <section class="fast-frame-section" id="adaptive-ui">
+          <div class="section-content">
+            <site-section-header class="fast-frame-header">
+              <fast-badge slot="badge" class="section-badge">FLEXIBLE AND ADAPTABLE</fast-badge>
+              <h2 class="section-heading">Dynamic themes with Adaptive UI</h2>
+              <p slot="body">FAST provides an innovative theming system called Adaptive UI, which builds design system properties that designers use every day directly into every component.</p>
+              <fast-button slot="action" appearance="accent">Explore components</fast-button>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <fast-frame></fast-frame>
+          </div>
+        </section>
+
+        <section class="section" id="integration">
+          <div class="section-content">
+            <site-section-header>
+              <fast-badge slot="badge" class="section-badge">FULLY INTEGRATED</fast-badge>
+              <h2 class="section-heading">Works with existing frameworks</h2>
+              <p slot="body">Standards-based Web Components are the foundation of each FAST component, making them compatible with almost any modern web framework.</p>
+              <fast-button slot="action" appearance="accent">Learn More</fast-button>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <site-content-placement-container section="framework"></site-content-placement-container>
+          </div>
+        </section>
+
+        <section class="section" id="features">
+          <div class="section-content">
+            <site-section-header>
+              <fast-badge slot="badge" class="section-badge">A COMPREHENSIVE TOOLKIT</fast-badge>
+              <h2 class="section-heading">Features and utilities</h2>
+              <p slot="body">There are a million and one great ways to build your next website or application. To support the technologies you choose, creating unopinionated code is at the center of every decision we make in FAST.</p>
+              <fast-button slot="action" appearance="accent" href="/docs/en/contributing/install">Read documentation</fast-button>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <site-content-placement-container section="feature"></site-content-placement-container>
+          </div>
+        </section>
+
+        <section class="section community-section" id="community">
+          <div class="section-content">
+            <site-section-header>
+              <fast-badge slot="badge" class="section-badge">LEARN, MODIFY AND FOLLOW</fast-badge>
```

Comment:
Could MODIFY be changed to CONTRIBUTE?  Would it improve the message that's trying to be conveyed?